app/models/Countdown.scala

```diff
@@ -0,0 +1,13 @@
+package models
+
+case class CountdownTheme(
+  backgroundColor: String,
+  foregroundColor: String,
+)
+
+case class CountdownSettings(
+  label: String,
+  countdownStartInMillis: String,
```

Comment:
I don't think this name is correct?
e.g. from CODE `countdownStartInMillis: "2025-05-13T10:43:31"`

Reply:
Oh , I used the same  values as we had used in support frontend before,  but  yes, it's misleading.I will update the names in both PRs ,thanks